src/componentsCommon/chackraInputFilterWithClear/chackraInputFilterWithClear.tsx

```diff
@@ -16,6 +16,10 @@ interface InputWithClearProps {
   title: string;
   propName: string;
 }
+export interface InputHeaderWithClearProps {
```

Comment:
shouldn't this extend the whole InputProps to pass down the others option (ie. onFocus, etc.)?